plugins/plugin-chart-echarts/src/Graph/constants.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { GraphSeriesOption } from 'echarts';
+
+export const DEFAULT_GRAPH_SERIES_OPTION: GraphSeriesOption = {
+  zoom: 1,
```

Comment:
I noticed that this might be better at something lower; see the 0.7 (upper) vs the current default of 1.0 (lower). This applies especially to circular mode, but it appears to also be over-zoomed on force mode.
![image](https://user-images.githubusercontent.com/33317356/108197149-a79f6500-7122-11eb-9d96-39dc1b467fda.png)
![image](https://user-images.githubusercontent.com/33317356/108197102-948c9500-7122-11eb-8ab4-86ed9f6f44f2.png)